src/App.tsx

```diff
@@ -135,7 +135,8 @@ function App() {
         if (!trajectory) {
             return null;
         }
-        return new BindingSimulator(trajectory, viewportSize.width / 5);
+        const longestAxis = Math.max(viewportSize.width, viewportSize.height);
+        return new BindingSimulator(trajectory, longestAxis / 3);
```

Comment:
I originally expected the longest axis to be the width, but you can make the window narrow and tall and get weird sizing